### ISSUE TITLE
fdp-play error

### ISSUE THREAD

**Issue by TheBojda** [fairDataSociety/fdp-play#140]
If I run fdp-play by any command like `fdp-play help`, it throws:

```
TypeError: Cannot set property crypto of #<Object> which has only a getter
    at 479 (/usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:1352913)
    at r (/usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:1621580)
    at /usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:1622285
    at /usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:1622508
    at /usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:1622515
    at /usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:81
    at Object.<anonymous> (/usr/local/lib/node_modules/@fairdatasociety/fdp-play/dist/index.js:3:237)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)

Node.js v20.13.1
```

**Comment by Aviksaikat** [comment]
use `node 18`.

**Comment by TheBojda** [comment]
I've sent a PR that fixes this. https://github.com/fairDataSociety/fdp-play/pull/141

**Comment by nugaon** [comment]
fixed in #141
